Integrations/QRadar/QRadar.yml

```diff
@@ -1503,6 +1503,76 @@ script:
     - contextPath: QRadar.Domains.TenantID
       description: ID of the Domain tenant.
       type: Number
+  - arguments:
+    - default: false
+      description: The name of the reference set to add or update a value in. If you
+        would like to create a new reference set you will  need to set element type
+      isArray: false
+      name: ref_name
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: 'The element type for the values allowed in the reference set.
+        You need to use it only for creating new reference set. The allowed values
+        are: ALN (alphanumeric), ALNIC (alphanumeric ignore case), IP (IP address),
+        NUM (numeric), PORT (port number) or DATE. Note that date values need to be
+        represented in milliseconds since the Unix Epoch January 1st 1970.'
+      isArray: false
+      name: element_type
+      predefined:
+      - ALN
+      - ALNIC
+      - IP
+      - NUM
+      - PORT
+      - DATE
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: The allowed values are "FIRST_SEEN", LAST_SEEN and UNKNOWN. The
```

Comment:
![image](https://user-images.githubusercontent.com/50294648/73138022-b70ac680-4066-11ea-8613-6faae8f71c43.png)

Please follow the templates for 'List of predetermined options'

Reply:
done